Utilities/rANS/include/rANS/SymbolStatistics.h

```diff
@@ -84,15 +84,14 @@ class SymbolStatistics
 
  private:
   void buildCumulativeFrequencyTable();
-
+  auto getCumulativeFrequency(size_t i) const { return mCumulativeFrequencyTable[i + 1] - mCumulativeFrequencyTable[i]; }
```

Comment:
my naming ``getCumulativeFrequency`` was a bad choice, this is actually ``getFrequency``